backend/src/packages/journal-entries/journal-entry.controller.ts

```diff
@@ -109,23 +178,135 @@ class JournalEntryController extends BaseController {
    *    get:
    *      description: Get all a journal entries
    *      responses:
+   *        201:
+   *          description: Successful operation
+   *          content:
+   *            application/json:
+   *              schema:
+   *                type: object
+   *                properties:
+   *                  journalEntry:
+   *                    $ref: '#/components/schemas/Journal Entry'
+   *        401:
+   *          description: Unauthorized
+   *          content:
+   *            application/json:
+   *              schema:
+   *                $ref: '#/components/schemas/Error'
+   *              example:
+   *                message: "Incorrect credentials."
+   *                errorType: "AUTHORIZATION"
+   */
+
+  private async getAll(
+    options: APIHandlerOptions<{ user: UserAuthResponseDto }>,
+  ): Promise<APIHandlerResponse> {
+    const { id } = options.user;
+
+    return {
+      status: HTTPCode.OK,
+      payload: await this.journalEntryService.findAllByUserId(id),
+    };
+  }
+
+  /**
+   * @swagger
+   * /journal-entries/:id:
+   *    get:
+   *      description: Get journal entry by id
+   *      responses:
+   *        201:
+   *          description: Successful operation
+   *          content:
+   *            application/json:
+   *              schema:
+   *                type: object
+   *                properties:
+   *                  journalEntry:
+   *                    $ref: '#/components/schemas/Journal Entry'
```

Comment:
I guess it should look like this. Please correct me if it shouldn't
```suggestion
   *              schema:
   *                $ref: '#/components/schemas/Journal Entry'
```